### PR TITLE
ts: add temporary standalone sdk importable from git

### PR DESCRIPTION
**Pull request by sipsma**
This will allow users to workaround the lack of pinning of tsx in previous engine releases until our next release.

### CODE REVIEW

File: sdk/typescript/standalone/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/dagger.gen.go
+/internal/dagger
+/internal/querybuilder
+/internal/telemetry
```
Comment (TomChv) [inline sdk/typescript/standalone/.gitignore:4]: Why the files in `internal` are pushed if they are ignored by this files?

**Review by TomChv — COMMENTED** [review]
Thanks for the changes! How the user can use this runtime though?

Can you show me an example?

File: sdk/typescript/standalone/.gitignore (hunk 1 above)
Comment (sipsma) [reply sdk/typescript/standalone/.gitignore:4]: I probably added them before adding the .gitignore. I'm gonna leave them in so we don't have weird linter complaints.

**Comment by sipsma** [comment]
> Thanks for the changes! How the user can use this runtime though?
> 
> Can you show me an example?

Going to include instructions in https://github.com/dagger/dagger/pull/7584 once this is merged

The tl;dr is the example in discord earlier: https://discord.com/channels/707636530424053791/1003718839739105300/1248698818212532316

File: sdk/typescript/standalone/.gitignore (hunk 1 above)
Comment (TomChv) [reply sdk/typescript/standalone/.gitignore:4]: Okay!

**Review by TomChv — APPROVED** [review]

**Pull request merged**